__tests__/models.test.js

```diff
@@ -99,6 +99,133 @@ describe("GET /api/reviews", () => {
         expect(shouldBe0.comment_count).toBe("0");
       });
   });
+  describe("?query", () => {
+    describe("category", () => {
+      test("200: features a category query, that selects the reviews by the specified category", () => {
+        return request(app)
+          .get("/api/reviews?category=social deduction")
+          .expect(200)
+          .then(({ body: { reviews } }) => {
+            expect(reviews).toHaveLength(11);
+            reviews.forEach((review) => {
+              expect(review).toHaveProperty("category", "social deduction");
+            });
+          });
+      });
+      test("200: endpoint responds with all the reviews, when query is omitted", () => {
+        return request(app)
+          .get("/api/reviews")
+          .expect(200)
+          .then(({ body: { reviews } }) => {
+            expect(reviews).toHaveLength(13);
+          });
+      });
+      //is this necessary, as it is done in the original get request right?
+      describe("Error Handling:", () => {
+        test("400: category doesn't exist", () => {
+          return request(app)
+            .get("/api/reviews?category=card game")
+            .expect(400)
+            .then(({ body: { msg } }) => {
+              expect(msg).toBe("Bad Request: Category does not exist!");
+            });
```

Comment:
Should this be a 400 🤔, could this category exist in the future? if so is there a more appropriate status that should be used instead?